producers-commissioning/src/main/resources/soapui/SDT-Endpoint-v2-soapui-project.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is a project for SoapUI which is an external piece of software.  It's not needed to build or run the application so doesn't need to be in the resources directory.

Reply:
better to keep all xml, properteis, yml files under resources